source/github-helpers/search-query.ts

```diff
@@ -120,6 +120,6 @@ export default class SearchQuery {
 	}
 
 	includes(...searchStrings: string[]): boolean {
-		return this.getQueryParts().some(queryPart => searchStrings.includes(queryPart));
+		return this.getQueryParts().some(queryPart => searchStrings.includes(queryPart) || searchStrings.includes(queryPart.replace(/"/g, '')));
```

Comment:
This is the wrong behavior and the test below should be dropped too. This suggestion was not meant to be part of SearchQuery, but honestly I can't think of another way to do this:

```ts
const isBugsPage = new SearchQuery(location.search).includes(`label:${await getBugLabel() ?? 'bug'}`);
```

without:

- adding support for quoted properties in `SearchQuery`
- adding a "quoter" helper to easily wrap `getBugLabel`, like:
	```ts
	`label:${SearchQuery.escapeValue(await getBugLabel() ?? 'bug'))}`
	```

`SearchQuery.escapeValue` is really simple:

```ts
static escapeValue(value: string) {
	return /:\s/.test(value) ? `"${value}"` : value;
}
```

This PR cannot be merged as is. It's probably best to take care of this in separate PR and then  we can come back here

Reply:
Curious why does it have to be part of SearchQuery?

Reply:
They both affect the search query, right?

Reply:
Sorry I was not clear.

My question was why can it not just be an export from that file just like we do with the `api.escapeKey`?


https://github.com/sindresorhus/refined-github/blob/f908677216e84f4a238a69f0573a257945ea6a25/source/features/release-download-count.tsx#L27

Reply:
In practice there's no difference since there's no tree-shaking. Since SearchQuery is structured as a class I think it's easier to have a `SearchQuery.escapeValue` method instead of exporting `escapeValue` on its own, which wouldn't make sense and would have to be named `escapeSearchQueryValue` 🤷‍♂️ 😅

Reply:
Got it